components/ImprovedSearch/PaginationItem.tsx

```diff
@@ -1,43 +1,37 @@
-import { usePagination } from 'react-instantsearch-hooks-web'
-import { Button } from "./Button"
-import styled from 'styled-components'
+import { Button, List, ListItemProps } from "@equinor/eds-core-react"
+import { usePagination } from "react-instantsearch-hooks-web"
+import styled from "styled-components"
 
-
-const StyledListItem = styled.li`
-  display: inline-block;
-`
-
-const PaginationLink = styled(Button)<{ isCurrent?: boolean; inverted?: boolean }>`
+const PaginationLink = styled(Button)<{ isCurrent?: boolean }>`
   width: 44px;
   height: 44px;
-  color: ${(props) => (props.inverted ? 'hsl(184, 31%, 74%)' : 'hsl(206, 32%, 21%);')}
-  ${({ isCurrent, inverted }) =>
+  color: ("hsl(206, 32%, 21%);")}
```

Comment:
This one was weird, you should'nt need the (" and prefer EDS token. You can also ask Espen if you don't know which colour it should be